docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,496 @@
+{{% admonition type="note" %}}
+You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).
+{{% /admonition %}}
+-->
+<!-- Add an image, GIF or video  as below
+
+{{< figure src="/media/docs/grafana/dashboards/WidgetVizSplit.png" max-width="750px" caption="DESCRIPTIVE CAPTION" >}}
+
+Learn how to upload images here: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets
+-->
+
+## Dashboards and visualizations
+
+### Flame graph improvements
+
+_Generally available in all editions of Grafana_
+
+<!-- Andrej Ocenas -->
+
+We've added four new features to the Flame graph visualization:
+
+- **Sandwich view**: You can now show a sandwich view of any symbol in the flame graph. Sandwich view shows all the callers on the top and all the callees of the symbol on the bottom. This is useful when you want to see the context of a symbol.
+- **Switching color scheme**: You can now switch color scheme between a color gradient based on the relative value of a symbol or by the package name of a symbol.
+- **Switching symbol name alignment**: Symbols with long names may be hard to differentiate if they have the same prefix. This new option allows you to align the text to the left or right so that you can see the part of the symbol name that's important.
+- **Improved navigation**: You can also highlight a symbol or enable sandwich view for a symbol from the table. Also, a new status bar on top of the flame graph displays which views are enabled.
+
+{{< video-embed src="/media/docs/grafana/panels-visualizations/screen-recording-grafana-10.1-flamegraph-whatsnew.mp4" >}}
+
+### Distinguish widgets from visualizations
+
+<!-- Alexa Vargas, Juan Cabanas -->
+
+_Experimental in all editions of Grafana_
+
+This experimental feature introduces a clear distinction between two different categories of panel plugin types: visualization panels that consume a data source and a new type, called _widgets_, that don't require a data source.
+
+Now, you can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on whether you've selected a widget or visualization, providing a streamlined editing experience.
+
+To see the widget editor in Grafana OSS or Enterprise, enable the `vizAndWidgetSplit` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
+
+{{< figure src="/media/docs/grafana/dashboards/WidgetVizSplit.png" max-width="750px" caption="New widget option added to empty dashboards" >}}
+
+### Transformations redesign
+
+<!-- Catherine Gui -->
+
+The transformations tab has an improved user experience and visual redesign. Now, you can explore transformations with categories and illustrations.
```

Comment:
@catherineymgui - I found the second sentence a bit unclear. Can we go with this? 

**Note, please don't commit any changes without checking in with Quynton first.**

```suggestion
The transformations tab has an improved user experience and visual redesign. Now, transformations are categorized, and each transformation type has an illustration to help you choose the right one.
```

Reply:
This sounds good to me.